build/ci/evergreen.yml

```diff
@@ -224,6 +238,18 @@ functions:
           <<: *go_options
         silent: true
         binary: build/package/set-up-gon.sh
+  "check license header":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        include_expansions_in_env:
+          - workdir
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          <<: *go_options
+        binary: build/package/generate-notices.sh
```

Comment:
[q] I am not sure who does the check. To me `generate-notices.sh` doesn’t check anything